src/components/AddItem.js

```diff
@@ -3,10 +3,12 @@ import { writeToFirestore } from '../lib/firebase';
 
 const AddItem = () => {
   let [name, setName] = useState('');
+  const token = localStorage.getItem('token');
```

Comment:
This is super nice of you. You should let the other team know. 🥇

Reply:
@TianaJohnson @laurenyz We updated the addItem function to include the updated collection name based on the user's token! Let @QMaximillian or I know how if you think this creates any conflict with your updated AddItem form.